videos/tasks.py

```diff
@@ -133,7 +136,31 @@ def delete_s3_objects(
 @app.task(acks_late=True)
 def update_transcripts_for_video(video_id: int):
     """Update transcripts for a video"""
-    threeplay_api.update_transcripts_for_video(Video.objects.get(id=video_id))
+    video = Video.objects.get(id=video_id)
+    if threeplay_api.update_transcripts_for_video(video):
+        website = video.website
+        if is_ocw_site(website):
+            search_fields = {}
+            search_fields[
+                "metadata__" + settings.FIELD_FILETYPE.replace(".", "__")
+            ] = "Video"
+            search_fields[
+                "metadata__" + settings.YT_FIELD_ID.replace(".", "__")
```

Comment:
Now that this is being done in a few places (including [similar line here](https://github.com/mitodl/ocw-studio/blob/86ea7e04b3633b92ed8b15255583c18857cf635a/videos/youtube.py#L278)), might be worth creating a separate function, something like:

```python
def get_metadata_field(sub_field):
    return "metadata__" + sub_field.replace(".", "__")
```